external/fv3viz/fv3viz/_plot_cube.py

```diff
@@ -59,15 +68,21 @@ def plot_cube(
     coastlines_kwargs: dict = None,
     **kwargs,
 ):
-    """ Plots tiled cubed sphere grids onto a global map projection
+    """ Plots an xr.DataArray containing tiled cubed sphere gridded data
+    onto a global map projection, with optional faceting of additional dims
 
     Args:
-        plottable_variable (xr.Dataset):
-            Dataset containing variable to plotted via pcolormesh, along with
-            coordinate variables (lat, latb, lon, lonb). This dataset object
-            can be created from the helper function :py:func:`mappable_var`, which takes
-            in an fv3gfs restart or diagnostic dataset along with the name of
-            the variable to be plotted.
+        ds (xr.Dataset):
```

Comment:
nit: docstrings shouldn't include types if there are type hints in the call signature